app/services/duplicate_profile_resolver.rb

```diff
@@ -54,8 +54,9 @@ def grouped_nonunique_profile_tuples
 
     # rubocop:disable Rails/SkipsModelValidations
     def migrate_profile_hosts(existing_p, duplicate_p)
-      duplicate_p.profile_hosts.where.not(host: existing_p.hosts)
-                 .update_all(profile_id: existing_p.id)
+      duplicate_p.profile_hosts.where.not(
```

Comment:
Observation: We no longer use `profile_hosts`.

Reply:
This resolver will not be used. It was a single time migration before we added uniqueness constraints in the DB. We can remove the service completely without affecting existing workflows.